### PR TITLE
PolicyInsights remediation + tracked resources APIs

### PR DESCRIPTION
**Pull request by pilor**
## Description

Generates the .NET SDK for the Microsoft.PolicyInsights remediations and tracked resources APIs.  These are two new resource types exposed in the Microsoft.PolicyInsights RP.  The existing resource types have not been modified.

The swagger PR for these new APIs was here:  https://github.com/Azure/azure-rest-api-specs/pull/3976
---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [x] Please add REST spec PR link to the SDK PR
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

**Comment by pilor** [comment]
@bulentelmaci @cemheren @eladperets FYI

File: src/SDKs/PolicyInsights/Management/Management.PolicyInsights/Generated/Models/QueryFailureError.cs
Comment (dsgouda) [inline src/SDKs/PolicyInsights/Management/Management.PolicyInsights/Generated/Models/QueryFailureError.cs]: This is a breaking change for existing customers, please bump major version number

**Review by dsgouda — CHANGES_REQUESTED** [review]

File: src/SDKs/PolicyInsights/Management/Management.PolicyInsights/Generated/Models/QueryFailureError.cs
Comment (pilor) [reply src/SDKs/PolicyInsights/Management/Management.PolicyInsights/Generated/Models/QueryFailureError.cs]: Done.  This model is a response model only used for error responses so it should be very unlikely that someone was setting the properties that are now correctly marked as readonly.

Comment (dsgouda) [reply src/SDKs/PolicyInsights/Management/Management.PolicyInsights/Generated/Models/QueryFailureError.cs]: Agreed, I'd take a second opinion from @shahabhijeet before approving

**Review by dsgouda — APPROVED** [review]
LGTM

**Comment by dsgouda** [comment]
Will merge once CIs pass

**Comment by dsgouda** [comment]
@azuresdkci Retest this please

**Pull request merged**
